server/utils/errorMessages.ts

```diff
@@ -19,6 +21,28 @@ export default {
     invalidDate: 'The date by which the service needs to be completed must be a real date',
     mustBeInFuture: 'The date by which the service needs to be completed must be in the future',
   },
+  reportingDate: {
+    from: {
+      dayEmpty: 'The "from" date must include a day',
+      monthEmpty: 'The "from" date must include a month',
+      yearEmpty: 'The "from" date must include a year',
+      invalidDate: 'The "from" date must be a real date',
+      mustBeAfterDayOne: 'Data before 01 June 2021 is not available',
+      mustBeBeforeToDate: 'The "from" date must be before the "to" date',
+      mustNotBeInFuture: `Data after ${DateUtils.getDateStringFromDateTimeString(
```

Comment:
I'm not quite sure how this works in JS, but isn't this whole exported object only evaluated once, when the server is started? So this would always return the date that the server was started.